src/modules/details/components/EditTicketForm.tsx

```diff
@@ -80,13 +85,21 @@ export const EditTicketForm = ({
   }, [usersList]);
 
   const handleUpdateTicket = useCallback(
-    ({ department_id, category_id, resolver_id, reason_for_update, status }) => {
+    ({
+      department_id,
```

Comment:
No need to destructure it as its been used directly.
`eg const handleUpdateTicket = useCallback((ticketPayload: IEditTicketPayload) => {}`